### PR TITLE
Install updates only when --updates option is specified

### LINKED ISSUE #21: Remove dnf/yum update from Dockerfiles
We should remove any sort of `dnf/yum update` from the Dockerfiles. One of the concerns is that parent images don't get updated fast enough, leaving stale packages in images. However, doing a yum update for every container slows down significantly creating a new instance of osbs-box.

### PR DESCRIPTION
**Pull request by vrutkovs**
Updates-testing repo is enabled in docker-compose.yml via build args.

It seems to build faster from scratch - ~3 min vs. ~9 mins. After the latyers 
are cached the time to build the images is approx. the same - around a minute 
and a half (due to setting up a new cluster mostly).

Fixes #21

### CODE REVIEW

File: client/Dockerfile [hunk 1]
```diff
@@ -1,11 +1,14 @@
 FROM fedora:25
 MAINTAINER Luiz Carvalho <lucarval@redhat.com>
 
+ARG UPDATES
 ARG UPDATES_TESTING
 
-RUN dnf update -y && dnf install -y dnf-plugins-core && \
-    if [ $UPDATES_TESTING ]; then dnf config-manager --set-enable updates-testing; fi && \
-    dnf -y update && \
+RUN if [ $UPDATES ]; then dnf update -y; fi && \
```
Comment (lcarva) [inline client/Dockerfile]: Hey! This same block structure can be used to install differently named packages for different distros, thus sharing the same Dockerfile :)

Ok, this comment is unrelated to this PR

**Comment by lcarva** [comment]
Merge conflicts.

File: client/Dockerfile (hunk 1 above)
Comment (vrutkovs) [inline client/Dockerfile]: Yep, I was thinking we should be building a base osbs-box image first and then base other on top of it. It would simplify a lot of things

**Comment by vrutkovs** [comment]
Fixed
